vernac/assumptions.ml

```diff
@@ -165,6 +165,28 @@ let label_of = let open GlobRef in function
   | ConstructRef ((kn,_),_) -> MutInd.label kn
   | VarRef id -> Label.of_id id
 
+let fold_with_full_binders g f n acc c =
```

Comment:
Couldn't we put this in termops so as to factor the econstr version?

Reply:
This one operates on Constr, not EConstr.

Reply:
That worked fine before too

Reply:
It would need to be in minievd because otherwise we need to cast all the types here and there, and it's a nightmare. I'd rather have this small code duplication than introduce strong coupling and dubious API in minievd.

Reply:
Having it in `vernac` looks late. `Termops` is already hybrid `constr`/`econstr`. Maybe having `termops` and `etermops` would clarify things (or `constrops` and `econstrops`) (or `termops` for `constr` and `econstrops` for `econstr`).

Reply:
@ppedrot wdyt?

Reply:
This function is only used there and clearly a specific need. I don't think it's worthwhile to export it for Constr.t anyways, you can always go through EConstr.t.

Reply:
What about the question of splitting `termops.ml`?